pkg/services/schemaloader/schemaloader.go

```diff
@@ -73,6 +76,21 @@ func (rs *SchemaLoaderService) DashboardApplyDefaults(input *simplejson.Json) (*
 	return output, nil
 }
 
+func (rs *SchemaLoaderService) GetJsonSchema(fn string) (*simplejson.Json, error) {
+	requestedFilePath := filepath.Join("pkg", "services", "schemaloader", "data", fn)
```

Comment:
Not sure this would work in a production build/release package of Grafana. Might want to consider either embedding or reading from a certain directory already part of release package. Or add a build step that would make sure to create/move the JSON schemas to the right directory. 

For example, the public directory might be interesting where we already have all client side code, email templates, views, dashboards (default home dashboard). Everything below public would be accessible from the browser though, but this is not any sensitive information

Reply:
Yup, absolutely need to use the new Go 1.16 embedding here.